### ISSUE TITLE
[App config file docs] Fix valid values example

### ISSUE THREAD

**Issue by fhightower** [ThreatConnect-Inc/threatconnect-developer-docs#274]
There is a code snippet here: https://docs.threatconnect.com/en/latest/deployment_config.html#external-variables that contains: 

```
"validValues": ["{USER:TEXT}", "${ORGANIZATION:TEXT}"]
```

This needs to be changed to:

```
"validValues": ["${USER:TEXT}", "${ORGANIZATION:TEXT}"]
```
(adding a `$` in from of `{USER:TEXT}`)

**Comment by Raymc007** [comment]
@fhightower this issue is fixed with PR #275.

**Comment by fhightower** [comment]
Thanks!
